detox/src/devices/drivers/android/emulator/EmulatorDriver.js

```diff
@@ -26,11 +27,12 @@ class EmulatorDriver extends AndroidDriver {
   constructor(config) {
     super(config);
 
-    this.freeDeviceFinder = new FreeEmulatorFinder(this.adb, this.deviceRegistry);
     const emulatorExec = new EmulatorExec();
     this._emuVersionResolver = new EmulatorVersionResolver(emulatorExec);
     this._emuLauncher = new EmulatorLauncher(emulatorExec);
-    this._avdValidator = new AVDValidator(emulatorExec);
+    this._avdsResolver = new AVDsResolver(emulatorExec);
```

Comment:
Was there any specific reason to pollute `this` with a never used `_avdsResolver` private property?

Reply:
Of course - will double check!